### DISCUSSION TITLE
Malformed Ansi Errors when trying new theme

### DISCUSSION THREAD

**Discussion by rojebd** [helix-editor/helix#9719] [General]
I'm trying to create a new theme but for some reason the colors don't work at all I have double checked and there is nothing bad with it but switching to a different theme works just fine?

[Helix Log](https://pastebin.com/YBQwtkTz) - (Verbosity On)

**Comment by kirawi** [comment]
Can you share the theme file?

**Reply by rojebd** [reply]
Oh sorry I forgot [here](https://pastebin.com/pX4gGLeB) it is
